plugins/module_utils/turbo/server.py

```diff
@@ -222,14 +222,18 @@ class FakeStdin:
 
 async def run_as_lookup_plugin(data):
     errors = None
-    from ansible.module_utils._text import to_native
-
     result = None
+
     try:
         import ansible.plugins.loader as plugin_loader
```

Comment:
The imported utilities were already in a try-except block. However, I do not think anything will change in adding `from ansible.module_utils._text import to_native` in the except block.

Reply:
I'm not sure I understand your comment.  `from ansible.module_utils._text import to_native` wasn't in the existing try/except block. It needs to be in one, because if there's an exception we need to return it. We can't add it to the existing try/except block because that except clause uses the `to_native` function, which we can't use until it has been successfully imported.

Reply:
I meant that the other imports were already in the try-except block and I do not understand why we should include the to_native import in a try-except block. I do not think the errors originate from the to_native import honestly, but I might be wrong.

Reply:
This import needs to be in a try/except block for the same reason the the other imports need to be in one. Because this `run_as_lookup_plugin()` function is called from a subprocess detached from the original Ansible playbook process, exceptions will be lost unless they are returned to that process via the unix socket. You can test this by explicitly raising an `ImportError` (or any `Exception`) outside the try/except block here and then running something like a vmware moid lookup. The exception will not be returned and the parent process will wait for data from the socket until the `ghost_killer()` task unlinks the server socket and stops the asyncio loop. Then the playbook task will end with a JSON decode error because the client socket received no data, and the exception information is never returned to the user. Note: if you try this, right now `ghost_killer()` waits an hour for running jobs to complete, so you should also change the number of seconds [here](https://github.com/ansible-collections/cloud.common/blob/c53d27f0c5ed4dff488b1d7be93e830ba3d5aa9d/plugins/module_utils/turbo/server.py#L324C70-L324C70) from 3600 to something much smaller if you want to see the final outcome quickly.

I agree that this is probably not the cause of the bug reports we've seen, which is why I didn't link this PR to any  existing issues, but it should still be fixed.